New package: Colorfy v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Colorfy
- Repository: https://github.com/JuliaGraphics/Colorfy.jl
- Created by: @eliascarv
- Version: v0.1.0
- Commit: 257c3bdc829f3858c0ffe515dbf6a0a03c674c8f
- Reviewed by: @eliascarv
- Reference: https://github.com/JuliaGraphics/Colorfy.jl/commit/257c3bdc829f3858c0ffe515dbf6a0a03c674c8f#commitcomment-140610496
- Description: Colorfy Julia objects
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6997cf828f4155e5e3c62c049a94d6f6f425ce02680af98228292bfa6c5fdd66c937906dff00f90bde8438ae7760ae66dd1210fb4eeae86a1be89bed5d1381ec16a60551b73315ae12566b9a6b665c2512875285decc9597d3f7f26190022e313de8f637d7754c11dc308340fe1a9c648ea04bbc817c751bb269e424a2da81bd3fa23bb8a3d9e8187e8fb096e01fd299f53e58d12a086a9e1e12961e4afacd11b49701c699caeeb050106e568fda982bb -->

eliascarv commented:
I ask that this PR be merged manually as this is a package with a different purpose than Colors.jl and is in the same organization as it.

[noblock]

JuliaRegistrator committed:
New package: Colorfy v0.1.0

UUID: 03fe91ce-8ec6-4610-8e8d-e7491ccca690
Repo: https://github.com/JuliaGraphics/Colorfy.jl.git
Tree: 8e87c180419c61fda911c65985fe4091a2b22a99

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
